inventory-fusion/src/main/scala/com/normation/inventory/provisioning/fusion/FusionReportUnmarshaller.scala

```diff
@@ -76,11 +76,27 @@ class FusionReportUnmarshaller(
    * - remove leading/trailing spaces ;
    * - remove multiple space
    */
-  private def optText(n:NodeSeq) = n.text match {
+  private def optText(n:NodeSeq): Option[String] = n.text match {
     case null | "" => None
     case s => Some(s.trim().replaceAll("""[\p{Blank}]+"""," "))
   }
 
+  //extremelly specialized convert used for optionnal field only, that
+  //log the error in place of using a box
+  private[this] def convert[T](input: Option[String], conv: String => T): Option[T] = {
+    try {
+      input.map( s => conv(s))
+    } catch {
+      case ex: Exception => Failure(ex.getMessage, Full(ex), Empty)
```

Comment:
You said in comment, that you would log the error in place, put i see no log here...

And as it use an option, The failure will be converted to None so you lost the failure (maybe returning a None would be better then